<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraffLTimerWizardPage</class>
 <widget class="QWizardPage" name="TraffLTimerWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,5">
       <item>
        <widget class="QLabel" name="label_tl_count">
         <property name="text">
          <string>No. of Traffic lights: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_tl_count"/>
       </item>
       <item>
        <widget class="QLabel" name="label_cycle_time">
         <property name="text">
          <string>Cycle time (secs): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cycle_time"/>
       </item>
       <item>
        <widget class="QLabel" name="label_avg_use_count">
         <property name="text">
          <string>Average Use Count: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_avg_use_count"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
